### ISSUE TITLE
Solar neutron activator in end

### ISSUE THREAD

**Issue by Livefood1212** [mekanism/Mekanism#7162]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Issue description:
i put my solar neutron near my fission reactor 
i put line under it and connect to waste barrel
the solar is half fill with waste but still not processing
pls help me

#### Steps to reproduce:
1.
2.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 
**Mekanism:** 
**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[[gist](https://gist.github.com/) / [pastebin](https://pastebin.com/) / etc link here. Please make sure that it isn't set to expire.]

**Comment by pupnewfster** [comment]
TLDR the end doesn't have any skylight. From our FAQ on discord:
Q: Why don't my Solar Neutron Activators work in the end? I have seen videos of people using them in the end and they worked for them.
A: Among various changes we made in V10, we modernized the logic our Solar Neutron Activators use to be more like our solar panels. This means that they require skylight to operate which Mojang marks as something the end does not have. This might have been a change vanilla made since 1.12 or it might just be that we now are actually properly checking if the dimension has skylight but either way it means that Solar Neutron Activators do not work in the end.
